tests/e2e/tests/regression/workbench/autocomplete.e2e.ts

```diff
@@ -8,84 +8,87 @@ const myRedisDatabasePage = new MyRedisDatabasePage();
-        const script = await workbenchPage.queryInputScriptArea.textContent;
-        await t.expect(script.replace(/\s/g, ' ')).eql('JSON.ARRAPPEND key value ', 'Result of sent command exists');
-        //Check that hint with arguments are displayed
-        await t.expect(workbenchPage.monacoHintWithArguments.visible).ok('Hints with arguments are displayed')
-        //Remove hints with arguments
-        await t.pressKey('esc');
-        //Check no hints are displayed
-        await t.expect(workbenchPage.monacoHintWithArguments.visible).notOk('Hints with arguments are not displayed')
-        //Check that using shortcut “Ctrl+Shift+Space” hints are displayed
-        await t.pressKey('ctrl+shift+space');
-        await t.expect(workbenchPage.monacoHintWithArguments.visible).ok('Hints with arguments are displayed')
-    });
+    ];
+
+    // Type command
+    await t.typeText(workbenchPage.queryInput, command, { replace: true });
+    // Open the read more by clicking on the "ctrl+space" and check
+    await t.pressKey('ctrl+space');
+    await t.expect(await workbenchPage.monacoCommandDetails.exists).ok('The "read more" about the command is not opened');
+    for(const detail of commandDetails) {
+        await t.expect(await workbenchPage.monacoCommandDetails.textContent).contains(detail, `The ${detail} command detail is not displayed`);
+    }
+    // Close the command details
+    await t.pressKey('ctrl+space');
+    await t.expect(await workbenchPage.monacoCommandDetails.exists).notOk('The "read more" about the command is not closed');
+});
+test('Verify that user can see static list of arguments is displayed when he enters the command in Editor in Workbench', async t => {
+    const command = 'AI.SCRIPTEXECUTE';
+    const command2 = 'TS.DELETERULE ';
+
+    // Type the command in Editing area
+    await t.typeText(workbenchPage.queryInput, command, { replace: true });
+    // Check that no hints are displayed
+    await t.expect(workbenchPage.monacoHintWithArguments.visible).notOk('Hints with arguments are still displayed');
+    // Add space after the printed command
+    await t.typeText(workbenchPage.queryInput, `${command} `, { replace: true });
+    // Check that hint with arguments are displayed
+    await t.expect(workbenchPage.monacoHintWithArguments.visible).ok('Hints with arguments are not displayed');
+
+    await t.typeText(workbenchPage.queryInput, command2, { replace: true });
+    // Check that hint with arguments are displayed
+    await t.expect(workbenchPage.monacoHintWithArguments.visible).ok('Hints with arguments are not displayed');
+    // Remove hints with arguments
+    await t.pressKey('esc');
+    // Verify that user can close the static list of arguments by pressing “ESC”
+    await t.expect(workbenchPage.monacoHintWithArguments.visible).notOk('Hints with arguments are still displayed');
+});
+// test('Verify that user can close the static list of arguments by pressing “ESC”', async t => {
```

Comment:
?

Reply:
thanks for noticing, I forgot to delete some already merged tests